app/javascript/shared/components/Alert.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div :class="['alert', `alert-${level}`, 'alert-dismissible', 'fade', 'show']" role="alert">
+    <strong>{{ title }}:</strong> {{ message }}
+    <button type="button" class="close" data-dismiss="alert" aria-label="Close">
```

Comment:
This is quite neat. When you close it does it also remove it from the alerts array in Page.vue?